backend/src/main/java/com/wpz/rbs/service/ActivityService.java

```diff
@@ -94,6 +100,15 @@ public ActivityConflict getConflictsRoom(int roomId) throws ParseException {
     }
 
     public ActivityConflict resolveConflictsRoom(int roomId, List<String> activitiesId) throws ParseException {
+        activitiesId.forEach(activityId -> {
+            Optional<Activity> activity = activityRepository.findById(activityId);
+            if (activity.isPresent() && !activity.get().getIs_usos()) {
+                String reservationId = activity.get().getUrl();
+                Reservation reservation = reservationService.getById(Integer.parseInt(reservationId));
```

Comment:
We should also change the status of the reservation to declined as in the decline